src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,7 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int bucketVolume;
+    private String model;
```

Comment:
Consider using access modifiers for the 'model' field to encapsulate the internal state of the Excavator object.